lib/src/LocalizationProvider.tsx

```diff
@@ -35,7 +31,7 @@ LocalizationProvider.propTypes = {
   /**
    * Your component tree.
    */
-  children: PropTypes.node.isRequired,
+  children: PropTypes.node,
 } as any;
 
-export default withDefaultProps({ name: 'MuiPickersLocalizationProvider' }, LocalizationProvider);
+export default LocalizationProvider;
```

Comment:
Accidental changes?

Reply:
What problem does `withDefaultProps` solve? I have removed it as `LocalizationProvider` is already a global provider, it duplicates.